sdk/cosmosdb/cosmos/src/documents/DatabaseAccount.ts

```diff
@@ -97,5 +97,9 @@ export class DatabaseAccount {
 export interface Location {
   name: string;
   databaseAccountEndpoint: string;
-  unavailable?: boolean;
```

Comment:
Is removing this field not a breaking change?

Reply:
no

Reply:
@ealsur yes, the type is public, but we have updated all the references in our sdk.

Reply:
@topshot99 This is a breaking change. Updating all references in our sdk is not sufficient, since this is part of public API. If a someone is using `Location` in their code, it would break after this change. We should do a backward compatible change. One such approach could be - 
```
export interface Location {
  name: string;
  databaseAccountEndpoint: string;
  unavailable?: boolean;
  lastUnavailabilityTimestamp?: number;
}
``` Here the only change is `addition` of a property, which is backward compatible.